src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1556,6 +1556,8 @@ protected virtual bool HandleControlStateOnTouch(Touch touch)
         [EditorBrowsable(EditorBrowsableState.Never)]
         protected virtual void OnEnabled(bool enabled)
         {
+            EnabledChanged?.Invoke(this, EventArgs.Empty);
```

Comment:
I think this event need to be invoked at the end of funtion as we assign the ControlState in the line.